projects/optic/src/commands/capture/capture.ts

```diff
@@ -224,11 +223,13 @@ const getCaptureAction =
         }
       });
 
-    for await (const observation of observations) {
-      if (observation.kind === CaptureObservationKind.InteractionCaptured) {
-        console.log(`Captured ${observation.path}`);
-      }
+    for await (const har of harEntries) {
+      captures.addHar(har);
     }
+    await captures.writeHarFiles();
```

Comment:
This isn't technically necessary right now, but will let us reused cached requests + may be helpful for us to debug + get users to send us debug information